__DEFINES/subsystem.dm

```diff
@@ -55,3 +57,4 @@
 #define SS_DISPLAY_TICKER         -10
 #define SS_DISPLAY_UNSPECIFIED     0
 #define SS_DISPLAY_SUN             10
+#define SS_DISPLAY_OSC			   20
```

Comment:
the other two are `OS` while this is `OSC`, this is upsetting

Reply:
OSC since removed